packages/vanilla/src/components/horizontalrule/__stories__/horizontalrule.stories.js

```diff
@@ -0,0 +1,54 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, select, text } from '@storybook/addon-knobs';
+import wrapper from './templates/wrapper.template';
+import horizontalrule from '../horizontalrule.template';
+import '../../../../../styles/scss/components/horizontalrule/_horizontalrule.scss';
+import 'carbon-components/scss/globals/grid/_grid.scss';
```

Comment:
Do we want to import this from `@carbon/grid/scss/_grid.scss`?

Reply:
Changed it.